pkg/action/get_all.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package action
+
+import (
+	"helm.sh/helm/v3/pkg/release"
+)
+
+// GetAll is the action for checking a given release's all information.
+//
+// It provides the implementation of 'helm get all'.
+type GetAll struct {
+	cfg *Configuration
+
+	// Initializing Version to 0 will get the latest revision of the release.
+	Version int
+
+	//
+	StatusPrinterOptions
+}
+
+// NewGetAll creates a new GetAll object with the given configuration.
+func NewGetAll(cfg *Configuration) *GetAll {
```

Comment:
Why should we almostly make a copy of `pkg/action/get.go`, duplicate codes? could we try to move `GetAll ` to `pkg/action/get_all.go` ？

```yaml
type GetAll struct {
    Get
    StatusPrinterOptions
}
```

Reply:
Because only `get all` prints all status messages including description. Other sub-commands such as `get notes` only prints NOTES so `--show-desc` is meaningless.